tests/providers/google/marketing_platform/hooks/test_campaign_manager.py

```diff
@@ -40,8 +40,7 @@ def setUp(self):
         "campaign_manager.GoogleCampaignManagerHook._authorize"
     )
     @mock.patch(
-        "airflow.providers.google.marketing_platform.hooks."
-        "campaign_manager.build"
+        "airflow.providers.google.marketing_platform.hooks." "campaign_manager.build"
```

Comment:
Can you revert these changes because they make it difficult for cherry-pick these operators to another project/package/release/etc.?

Reply:
Done